benchmarks/build_test.go

```diff
@@ -21,10 +21,12 @@ import (
 )
 
 var (
-	baseImg     = "some-org/" + h.RandString(10)
-	trustedImg  = baseImg + "-trusted-"
-	builder     = "cnbs/sample-builder:bionic"
-	mockAppPath = filepath.Join("..", "acceptance", "testdata", "mock_app")
+	baseImg             = "some-org/" + h.RandString(10)
+	trustedImg          = baseImg + "-trusted-"
+	builder             = "cnbs/sample-builder:bionic"
+	mockAppPath         = filepath.Join("..", "acceptance", "testdata", "mock_app")
+	paketoBuilder       = "paketobuildpacks/builder:base"
```

Comment:
Could we make this configurable with an env var or something? It would be nice to make it easy for contributors to test this out with builders that they care about.

Reply:
Sure, I have created `setEnv()` to set these values. If value is not set in the environment, default values will be assigned. `setEnv()` will be invoked at first of `BenchmarkBuild()`. Does this treatment make sense?